src/main/java/com/monumental/services/MonumentService.java

```diff
@@ -154,19 +178,38 @@ private Predicate buildTagsQuery(CriteriaBuilder builder, CriteriaQuery query, R
      * @param distance - The distance from the comparison point to search in, units of miles
      * @param tags - List of tag names to search by
      * @param materials - List of material tag names to search by
+     * @param sortType - The way in which to sort the results by
      */
     private void buildSearchQuery(CriteriaBuilder builder, CriteriaQuery query, Root root, String searchQuery,
                                   Double latitude, Double longitude, Integer distance, List<String> tags,
-                                  List<String> materials, Boolean orderByResults) {
+                                  List<String> materials, SortType sortType) {
 
         List<Predicate> predicates = new ArrayList<>();
 
+        boolean sortByRelevance = false;
+        boolean sortByDistance = false;
+
+        switch (sortType) {
+            case NEWEST:
+                this.sortByDate(builder, query, root, true);
+                break;
+            case OLDEST:
+                this.sortByDate(builder, query, root, false);
+                break;
+            case DISTANCE:
+                sortByDistance = true;
```

Comment:
Missing `break;` here?

Reply:
Nevermind, you probably intentionally did it that way so searches with distance and text can be ordered by both.

Reply:
Fixed. Also let's not talk about the genius that is
```java
default:
    break;
```